### ISSUE TITLE
Onderzoek: Kunnen we Squit-dossiers zonder BAG-ID mappen naar dossiers met BAG-ID (obv zelfde adres)

### ISSUE THREAD

**Issue by Joost-vandijk** [coherenza/gebouwenpaspoort-webui#18]
Ze zijn er achter gekomen dat een aantal dossiers niet worden opgevoerd met een BAG-ID. Soms als foute registratie (luiheid van een inspecteur), maar soms omdat er voor nieuwbouwprojecten nog een BAG-ID bestaat. Adressen voor nieuwbouw worden ook als reeks gehanteerd, bijvoorbeeld Lamstraat 5-30, wat het zoeken/vinden bemoeilijkt.

Kunnen wij hierin iets bieden dat Squit-dossiers zonder BAG-ID alsnog op adres gemapt worden? Of zorgt dit weer voor allerlei andere problemen? In hun nieuwe systeem RX Mission is het gebruik van een BAG-ID verplicht/required field. Het probleem is dus oude, incomplete data. Vaak zijn er dan meerdere dossiers, waar er maar 1 een BAG-ID heeft. In deze gevallen zullen ze vaak wel hetzelfde adres hebben. Kunnen wij daarin een vlookup/join exercitie doen om deze dubbelingen te ontdubbelen (voor zover mogelijk)?

**Comment by importalis** [comment]
Ja, het is mogelijk om gestructureerde adresgegevens te mappen op een BAG-ID. 
Met slechter gestructureerde adressen hebben we minder goede ervaringen (zie Limburgs archief). Gestructureerd betekent dat straatnaam, huisnummer, postcode en woonplaats in afzonderlijke velden zijn opgenomen.

Zijn er voorbeelden waar dit probleem optreedt?

**Comment by Joost-vandijk** [comment]
Ja, ze kwamen tijdens het overleg al op meerdere dossiers zonder BAG-ID. Ik zal vragen of ze een lijst kunnen uitdraaien van deze dossiers. Daarbij hebben we nodig: Zaaknummer, Straatnaam, Huisnummer, Postcode en Woonplaats. Als ik wat binnen krijg, laat ik het weten.

Wat mij betreft is dit achterstallig database-onderhoud aan de gemeentekant, en pakken we dit op als er geen andere changes nodig zijn.
